nrfx/drivers/src/nrfx_twim.c

```diff
@@ -375,6 +375,26 @@ nrfx_err_t nrfx_twim_reconfigure(nrfx_twim_t const *        p_instance,
     return err_code;
 }
 
+void nrfx_twim_callback_get(nrfx_twim_t const *       p_instance,
+                            nrfx_twim_evt_handler_t * p_event_handler,
+                            void **                   pp_context)
+{
+    twim_control_block_t * p_cb  = &m_cb[p_instance->drv_inst_idx];
+
+    *p_event_handler = p_cb->handler;
+    *pp_context      = p_cb->p_context;
+}
+
+void nrfx_twim_callback_set(nrfx_twim_t const *     p_instance,
+                            nrfx_twim_evt_handler_t event_handler,
+                            void *                  p_context)
+{
+    twim_control_block_t * p_cb  = &m_cb[p_instance->drv_inst_idx];
+
+    p_cb->handler   = event_handler;
```

Comment:
I guess the interrupt should be disabled before the handler is changed, and afterwards, it should be enabled only if `event_handler` was not `NULL`:
https://github.com/zephyrproject-rtos/hal_nordic/blob/2c14cbfc7d6b6171307cb8bbc411395fae9a15d2/nrfx/drivers/include/nrfx_twim.h#L259

Reply:
fixed by allowing to reconfigure the handler only in the non-blocking mode